### PR TITLE
Add replication2_rt_sink_connection.

### PR DESCRIPTION
**Pull request by cmeiklejohn**
@seancribbs

### CODE REVIEW

File: db/seed.rb [hunk 1]
```diff
@@ -158,6 +158,7 @@ def create_riak_test(name, *args)
   create_riak_test 'mapred_http_errors', tags.merge('min_version' => '1.4.2')
   # Riak 1.4.4
   create_riak_test 'verify_2i_aae', tags.merge('min_version' => '1.4.4')
+  create_riak_test 'verify_2i_aae', tags.merge('min_version' => '1.4.4')
```
Comment (seancribbs) [inline db/seed.rb]: You've got two of these, did you not rebase/ff your branch?

Comment (cmeiklejohn) [inline db/seed.rb]: Weird, I rebased off master.  One moment.

**Comment by cmeiklejohn** [comment]
Fixed.

**Comment by seancribbs** [comment]
Looks good. :+1:

**Comment by seancribbs** [comment]
Merging and deploying.

**Comment by seancribbs** [comment]
Deployed!

**Pull request merged**